File: csv/jruby/create/csv_create.rb
#! /usr/bin/jruby
# -*- encoding: utf-8 -*-
#
#	create/csv_create.rb
#
#					Jul/02/2011
#
load '/var/www/data_base/common/ruby_common/text_manipulate.rb'
# ---------------------------------------------------------------------
def prepare_data_proc ()
	dict_aa = {}
	dict_aa=dict_append_proc(dict_aa,"t1271","千葉",17626,"2007-2-25")
	dict_aa=dict_append_proc(dict_aa,"t1272","勝浦",29819,"2007-7-27")
	dict_aa=dict_append_proc(dict_aa,"t1273","市原",51872,"2007-2-8")
	dict_aa=dict_append_proc(dict_aa,"t1274","流山",27615,"2007-1-21")
	dict_aa=dict_append_proc(dict_aa,"t1275","八千代",85162,"2007-5-24")
	dict_aa=dict_append_proc(dict_aa,"t1276","我孫子",21581,"2007-4-7")
	dict_aa=dict_append_proc(dict_aa,"t1277","鴨川",42693,"2007-4-8")
	dict_aa=dict_append_proc(dict_aa,"t1278","銚子",54197,"2007-8-15")
	dict_aa=dict_append_proc(dict_aa,"t1279","市川",28936,"2007-10-27")
#
	return	dict_aa
end
# ---------------------------------------------------------------------
puts	"*** 開始 ***"
text_file = ARGV[0]
#
dict_aa = prepare_data_proc()
#
dict_display_proc(dict_aa)
#
csv_write_proc(text_file,dict_aa)
#
puts	"*** 終了 ***"
#
# ---------------------------------------------------------------------

File: csv/rhino/create/csv_create.js
// ------------------------------------------------------------------
//
//	csv_create.js
//
//						Jul/06/2011
//
// ------------------------------------------------------------------
importPackage (java.io);

load ("/var/www/data_base/common/rhino_common/file_io.js");
load ("/var/www/data_base/common/rhino_common/text_manipulate.js");

var file_out=arguments[0];

print	("*** 開始 ***");

var dict_aa = data_prepare_proc ();

dict_display_proc (dict_aa);

csv_write_proc (file_out,dict_aa);

print	("*** 終了 ***");
// ------------------------------------------------------------------
function data_prepare_proc ()
{
	var dict_aa = new Object ();

	dict_aa["t1271"] = {"name": "千葉", "population": 85431, "date_mod": "1951-8-22"};
	dict_aa["t1272"] = {"name": "勝浦", "population": 57635, "date_mod": "1951-7-15"};
	dict_aa["t1273"] = {"name": "市原", "population": 32547, "date_mod": "1951-2-24"};
	dict_aa["t1274"] = {"name": "流山", "population": 69839, "date_mod": "1951-8-16"};
	dict_aa["t1275"] = {"name": "八千代", "population": 41834, "date_mod": "1951-9-8"};
	dict_aa["t1276"] = {"name": "我孫子", "population": 82836, "date_mod": "1951-3-25"};
	dict_aa["t1277"] = {"name": "鴨川", "population": 19832, "date_mod": "1951-4-19"};
	dict_aa["t1278"] = {"name": "銚子", "population": 93854, "date_mod": "1951-5-18"};
	dict_aa["t1279"] = {"name": "市川", "population": 37615, "date_mod": "1951-2-1"};

	return	dict_aa;
}

// ------------------------------------------------------------------
